pkg/user/tx_client.go

```diff
@@ -142,6 +153,8 @@ type TxClient struct {
 	signer   *Signer
 	registry codectypes.InterfaceRegistry
 	grpc     *grpc.ClientConn
```

Comment:
[nit] should this field be deprecated?

```suggestion
	// grpc is the primary GRPC connection.
	// Deprecated: use conns[0] instead.
	grpc     *grpc.ClientConn
```

Reply:
yeah no dangling code pls